relay-sampling/src/lib.rs

```diff
@@ -374,6 +374,15 @@ pub enum SamplingValue {
     Factor { value: f64 },
 }
 
+impl SamplingValue {
+    fn get_sampling_base_value(&self) -> f64 {
```

Comment:
nit: Our getters usually do not have a `get_` prefix (see also [rust docs](https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter)). Should we just call this `value()`?